src/components/ui/NavBar/index.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import styled from '@emotion/styled';
 
-import Icon from 'components/ui/Icon';
+import IconContainer from 'components/containers/IconContainer';
```

Comment:
We don't need a container for icons nor Navigations.

Reply:
Ok refactoring it asap

Reply:
All done